### ISSUE TITLE
Traceback when saving to AI compatible PostScript

### ISSUE THREAD

**Issue by jpfleury** [tzunghaor/inkscape-eps-export#3]
**Describe the bug**

Trying to save the attached SVG file as AI compatible PostScript results to traceback.

**Files**

[Starry dusk on Matterhorn - simple.svg.zip](https://github.com/tzunghaor/inkscape-eps-export/files/3487659/Starry.dusk.on.Matterhorn.-.simple.svg.zip) [1]

[1]: Source: https://forum.kde.org/viewtopic.php?f=312&t=160623#p416780

**To Reproduce**

1. Extract the attached zip file.
2. In Inkscape, go to "File > Open..." and select the extracted SVG file.
3. Go to "File > Save As..." and choose the AI compatible PostScript format.
4. See error:

```
Traceback (most recent call last):
  File "aieps_output.py", line 1103, in <module>
    print(converter.convert())
  File "aieps_output.py", line 1075, in convert
    self.walkElem(self.root)
  File "aieps_output.py", line 816, in walkElem
    self.walkElem(child)
  File "aieps_output.py", line 735, in walkElem
    uri, shortTag = elem.tag.split('}')
ValueError: need more than 1 value to unpack
```

**More information**

On Xubuntu 19.04 64-bit and Inkscape 0.92.4 (5da689c313, 2019-01-14).

**Comment by tzunghaor** [comment]
Thank you for the detailed error report. I'm a bit busy right now, I can start working on this next weekend.

**Comment by tzunghaor** [comment]
Fixed the crash in v1.0.3

However, the file you've attached makes heavy use of transparency, which is - as written in the readme - not supported by EPS, therefore the output of the script will look pretty different.

**Comment by jpfleury** [comment]
Great. Thanks a lot.
